hudi-cli/src/main/java/org/apache/hudi/cli/utils/HiveUtil.java

```diff
@@ -42,17 +41,8 @@
   private static Connection connection;
```

Comment:
May be we can just remove `connection` variable

Reply:
huh. you are right. its only used here.